### PR TITLE
refactor(types): add body-parser types

### PR DESCRIPTION
**Pull request by macklinu**
I am aiming to get familiar with this codebase and hope to contribute small things (adding types, fixing typos, etc.) as I navigate through the codebase and try to understand what's going on.

I was working on a small Express-based project of my own and learned that there are types for the body-parser package, and that was one of the first things I saw in `peril.ts`. 😄

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
   "author": "Orta",
   "license": "MIT",
   "dependencies": {
+    "@types/body-parser": "^1.16.4",
```
Comment (macklinu) [inline package.json]: Do we need to ship all the `@types` packages as dependencies, or can they be moved to devDependencies?

Comment (orta) [inline package.json]: Dep Deps is fine yeah

**Comment by orta** [comment]
👍

File: package.json (hunk 1 above)
Comment (orta) [inline package.json]: I just did it to be lazy, because no-one is going to use peril as a dependency

Comment (macklinu) [inline package.json]: Ah, that makes sense - didn't think through that one 👍

**Pull request merged**